examples/README.md

```diff
@@ -0,0 +1,12 @@
+# go-tpm usage examples
+
+This directory contains binaries demonstrating usage of go-tpm.
```

Comment:
[optional nit]: This sentence could be simplified and made clearer at the same time:

"This directory contains binaries that show how to use go-tpm."

Reply:
Done